ui/lib/pki/addon/routes/roles/index.js

```diff
@@ -5,13 +5,6 @@ import { hash } from 'rsvp';
 export default class PkiRolesIndexRoute extends PkiOverviewRoute {
   @service store;
   @service secretMountPath;
-  @service pathHelp;
-
-  beforeModel() {
-    // Must call this promise before the model hook otherwise
-    // the model doesn't hydrate from OpenAPI correctly.
-    return this.pathHelp.getNewModel('pki/role', this.secretMountPath.currentPath);
```

Comment:
ah, memories. I remember finding this breakthrough with you months ago. Sad to see it go, but the pattern you implemented seems more secure.